pkg/apis/pipeline/v1beta1/swagger.json

```diff
@@ -0,0 +1,2186 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Python SDK for Tekton Pipeline",
+    "title": "Tekton",
+    "version": "v0.1"
```

Comment:
This version is the same specified in https://github.com/tektoncd/pipeline/pull/3081/files#diff-03086a4a73c6f26ac1efba516c7ed86fR27?

It looks like it would make sense for it to be to set to a unique value, perhaps related to the git sha it was generated from? We could extend our release process to generate these with the specific version and publish it somewhere on the `tekton.dev` domain, so that it would be browsable per-version

Reply:
Updated to the latest release version. I think we cannot use the git sha, since if so, the `diff pkg/*` in CI will be failed, so I used the latest release tag `v0.16.1`. Personally that's not critical for now if just for Python SDK generation :-)